docs/contributing/local-development/Local_Linking.mdx

```diff
@@ -45,6 +46,24 @@ Now, you should be able to run ESLint in the local downstream repository as you
 To check that the local package is being used, consider adding a `console.log("Hello, world!");` to a file such as `./packages/eslint-plugin/dist/index.js` and making sure that log appears when linting the local downstream repository.
 :::
 
+## Testing Rules
+
+Now that you've linked the `@typescript-eslint/*` packages with your local downstream repository, the next step would be to include the rule on the local repository ESLint configuration file, e.g:
+
+```json
+{
+  "rules": {
+    "@typescript-eslint/your-awesome-rule": "error"
+  }
+  // etc
+}
+```
+
+After that, you just need to run your repository `lint` script.
+
+- `npm run lint`
+- `yarn lint`
```

Comment:
The problem with explicitly listing out "lint" scripts is that there are so many different ways folks can do it, it's hard to express all of them. What about `pnpm` users? What about folks who have different "lint" scripts?

Suggestion: instead of saying `lint` and then this list, use a more casual phrase like "your repository's lint script".

Reply:
Makes sense. Something already existing in our docs in one way is no guarantee that that's the right way 😄